examples/dump_media/README.md

```diff
@@ -0,0 +1,23 @@
+# Download all media from a Studio account
+
+This script downloads all media in a file / directory hierarchy:
```

Comment:
We should mention here that a media embedded in more than one course will be downloaded each time.